### PR TITLE
min width for 0 value set to 0

### PR DESCRIPTION
**Pull request by pandafulmanda**
## Before
screen shot 2015-09-16 at 1 21 26 pm

## After
screen shot 2015-09-16 at 1 21 37 pm

### CODE REVIEW

File: resources/styles/global/progress-bars.less [hunk 1]
```diff
@@ -7,3 +7,7 @@
   #gradient > .striped(rgba(255,255,255,.15), 135deg);
   background-size: 20px 20px;
 }
+
+.progress-bar[aria-valuenow="0"] {
```
Comment (philschatz) [inline resources/styles/global/progress-bars.less]: hmm, is it non-trivial to rely on a non-aria attribute?

Comment (pandafulmanda) [inline resources/styles/global/progress-bars.less]: oh hmm, i was simplying overriding something from bootstrap, but yeah we could do that

**Comment by philschatz** [comment]
Screenshot from demo:

image

**Comment by pandafulmanda** [comment]
just checked on that view bars stuff, this PR would fix that

File: resources/styles/variables/paths.less [hunk 2]
```diff
@@ -1,4 +1,4 @@
 @fa-font-path: './fonts';
 
-@bootstrap-path: '../../../node_modules/bootstrap/less/';
-@material-path: '../../../bower_components/bootstrap-material-design/less/';
+@bootstrap-path: '~bootstrap/less/';
```
Comment (philschatz) [inline resources/styles/variables/paths.less]: Cool, I didn't know about `~`. nice!

Comment (pandafulmanda) [inline resources/styles/variables/paths.less]: enabled by `less-loader` package :)

**Pull request merged**